src/Button/ButtonBase.tsx

```diff
@@ -70,6 +71,7 @@ const ButtonBase = forwardRef(
         data-block={block ? 'block' : null}
         data-size={size === 'small' || size === 'large' ? size : undefined}
         data-no-visuals={!LeadingVisual && !TrailingVisual && !TrailingAction ? true : undefined}
+        aria-disabled={inactive}
```

Comment:
I think the property `aria-disabled` needs to be removed entirely if the `inactive` value is `false`

Reply:
```suggestion
        aria-disabled={inactive ? true : undefined}
```

Agree with PK here